Sources/XcbeautifyLib/Renderers/OutputRendering.swift

```diff
@@ -580,30 +580,45 @@ extension OutputRendering {
     }
 
     func formatParallelTestCasePassed(group: ParallelTestCasePassedCaptureGroup) -> String {
+        let suite = group.suite
         let testCase = group.testCase
         let device = group.device
         let time = group.time
-        return colored ? Format.indent + TestStatus.pass.foreground.Green + " " + testCase + " on '\(device)' (\(time.coloredTime()) seconds)" : Format.indent + TestStatus.pass + " " + testCase + " on '\(device)' (\(time) seconds)"
+
+        return formatParallelTestCase(result: TestStatus.pass, resultColor: \.Green, suite: suite, testCase: testCase, device: device, time: time)
     }
 
     func formatParallelTestCaseSkipped(group: ParallelTestCaseSkippedCaptureGroup) -> String {
+        let suite = group.suite
         let testCase = group.testCase
         let device = group.device
         let time = group.time
-        return colored ? Format.indent + TestStatus.skipped.foreground.Yellow + " " + testCase + " on '\(device)' (\(time.coloredTime()) seconds)" : Format.indent + TestStatus.skipped + " " + testCase + " on '\(device)' (\(time) seconds)"
+
+        return formatParallelTestCase(result: TestStatus.skipped, resultColor: \.Yellow, suite: suite, testCase: testCase, device: device, time: time)
     }
 
     func formatParallelTestCaseAppKitPassed(group: ParallelTestCaseAppKitPassedCaptureGroup) -> String {
+        let suite = group.suite
         let testCase = group.testCase
         let time = group.time
-        return colored ? Format.indent + TestStatus.pass.foreground.Green + " " + testCase + " (\(time.coloredTime()) seconds)" : Format.indent + TestStatus.pass + " " + testCase + " (\(time)) seconds)"
+
+        return formatParallelTestCase(result: TestStatus.pass, resultColor: \.Green, suite: suite, testCase: testCase, device: nil, time: time)
     }
 
     func formatParallelTestCaseFailed(group: ParallelTestCaseFailedCaptureGroup) -> String {
+        let suite = group.suite
         let testCase = group.testCase
         let device = group.device
         let time = group.time
-        return colored ? "    \(TestStatus.fail.f.Red) \(testCase) on '\(device)' (\(time.coloredTime()) seconds)" : "    \(TestStatus.fail) \(testCase) on '\(device)' (\(time) seconds)"
+
+        return formatParallelTestCase(result: TestStatus.fail, resultColor: \.Red, suite: suite, testCase: testCase, device: device, time: time)
+    }
+
+    private func formatParallelTestCase(result: String, resultColor: KeyPath<String.StringForegroundColorizer, String>, suite: String, testCase: String, device: String?, time: String) -> String {
```

Comment:
The `TestCase` enum has static properties rather than cases with `rawValue`s. I switched against the static strings, because I expect converting the properties to cases will have larger impacts.
```swift
enum TestStatus {
    static let pass = "✔"
    static let fail = "✖"
    static let pending = "⧖"
    static let completion = "▸"
    static let measure = "◷"
    static let skipped = "⊘"
}
```

Reply:
Good call. I forgot that I swapped `TestStatus` to static properties.